src/components/webviews/CozyWebView.js

```diff
@@ -182,6 +222,14 @@ export const CozyWebView = ({
         tryConsole(m, log, logId)
         nativeIntent?.tryEmit(m)
 
+        const data = m?.nativeEvent?.data
+        if (data) {
+          const { message, value } = JSON.parse(data)
+          if (message === 'startLauncher') {
+            trySetLauncherContext({ state: 'launch', value })
+          }
+        }
```

Comment:
nit: It would be perfect if this is moved in an external method like it is done by `tryConsole(m,...), `tryNavigatorShare(m,...)` and `tryCrypto(m,...)`. But only if this is easy. Current implementation is good.

Reply:
Done